apps/07jquery-tic-tac-toe/script.js

```diff
@@ -2,4 +2,72 @@
 
 $(document).on('ready', function() {
   // Put app logic in here
-});
+
+  //sets playerTurn to X
+  var playerTurn = 'X';
+
+
+  //click datacell to run all the info in this function
+   $('[data-cell]').click(function() {
+        //ensuring the spot is open
+        if ($(this).text() === 'X' || $(this).text() === 'O') {
```

Comment:
Good data scrub. 

You could also write  `if ($(this).text()) { ..` for short since an empty string is considered falsey.